pages/glossary/context.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Context
+key: context
+---
+
+The context of a piece of information is any other information that helps a user understand it by clarifying its meaning.
+
+Both the [visual context](#visual-context) and the [programmatically determined context](#programmatically-determined-context) contribute to the context.
+
+**Note**: To be considered as context, content must be available similarly to all users with similar abilities. Thus, usually, content from another [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) is not part of the context as there is no guarantee that all users have access to it (the global topic of a website may nonetheless be considered context even if it's not repeated on every single page of the site).
```

Comment:
This isn't in the definition. You can use notes to clarify a definition, but you can't use them to introduce new ideas to it. The same is true for all of the other notes here.

Reply:
Definition removed.